README.md

```diff
@@ -2,6 +2,11 @@
 
 Swanky node is the Substrate based blockchain configured to enable `pallet-contracts` and more features to help Wasm Smart Contract development.
 
+## Features
+- pallet-contracts enabled
```

Comment:
Can you specify which pallet-contract version will it targets? And if `unstable-feature` is enabled by default